pkg/kotsadm/types/constants.go

```diff
@@ -13,6 +13,9 @@ const ExcludeValue = "true"
 const BackupLabel = "kots.io/backup"
 const BackupLabelValue = "velero"
 
+const TroubleshootKey = "troubleshoot.io/kind"
+const TroubleshootValue = "support-bundle"
```

Comment:
It feels like these labels are troubleshoot-specific and could be constants imported from the troubleshoot module. If it ever changes in troubleshoot, we may also have to change it here too so that it gets discovered.

Reply:
Good idea! I will try to see if I can implement it.

Reply:
I think that can be in a separate PR if troubleshoot doesn't export the labels as constants.  It may be something we have to wait for a new release to implement if that is the case.